PERFORMANCE.md

```diff
@@ -0,0 +1,343 @@
+# Performance Results
+
+The performance of SmartSim is detailed below across the Super Computer Horizon.
```

Comment:
We have collected SmartSim performance results on Horizon, a Cray XC50 supercomputer equipped with Nvidia P100 GPUs, 64 GB DDR4-2400 memory, and 18-core 2.3 GHz Intel Broadwell processors.

Reply:
You need to add the details of the system that I put here